pkg/baseline/packageIndex.json

```diff
@@ -1,14 +1,2219 @@
 {
   "Packages": {
```

Comment:
It should, but around the time when the dev/eng branch merged in, we stopped pushing baseline packages to the feed.  @ericstj was looking into bringing this back.